packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.tsx

```diff
@@ -1,17 +1,54 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useActiveWalletAccount, useAuthorize, useWalletAccountsList } from '@deriv/api';
-import { DesktopWalletsList, WalletsAddMoreCarousel, WalletsCarousel, WalletTourGuide } from '../../components';
+import {
+    DesktopWalletsList,
+    WalletsAddMoreCarousel,
+    WalletsCarousel,
+    WalletsResetMT5Password,
+    WalletTourGuide,
+} from '../../components';
+import { useModal } from '../../components/ModalProvider';
+import { getActionFromUrl } from '../../helpers/urls';
 import useDevice from '../../hooks/useDevice';
+import { platformMapping } from '../../utils/cfdUtils';
 import './WalletsListingRoute.scss';
 
 const WalletsListingRoute: React.FC = () => {
     const { isMobile } = useDevice();
     const { data: walletAccounts } = useWalletAccountsList();
     const { switchAccount } = useAuthorize();
     const { data: activeWallet } = useActiveWalletAccount();
+    const { show } = useModal();
+
+    const [password, setPassword] = useState('');
+    const resetTradingPlatformActionParams = getActionFromUrl();
 
     const firstLoginid = walletAccounts?.[0]?.loginid;
 
+    useEffect(() => {
+        let platformKey;
+        if (resetTradingPlatformActionParams !== null) {
+            platformKey = platformMapping[resetTradingPlatformActionParams];
+        }
```

Comment:
@aizad-deriv maybe better to use const instead of let and write like this:

`const platformKey = resetTradingPlatformActionParams ? platformMapping[resetTradingPlatformActionParams] : null;`